denizen_scripts/survival/repo_link/global_systems/economy_test.dsc

```diff
@@ -91,6 +91,28 @@ economy_withdraw:
     - else if <player.money> < <[amount]>:
       - narrate "<&c>You don't have enough money."
       - stop
+    - money take quantity:<[amount]>
     - give "<item[economy_bank_note].with[nbt=value/<[amount]>|lore=<&a>------------------------|<&e>Value<&co> <&a><[amount]>|<&e>Right click while holding to deposit.|<&a>------------------------]>"
     - narrate "<&b>You have withdrawn <&a><[amount].economy.formatted><&b> from your account."
     - narrate "<&b>Check your inventory for the bank note."
+
+economy_pay:
+  type: command
+  name: pay
+  script:
+    - define amount <context.args.second||null>
+    - define payee <context.args.first||null>
+    - if <[amount]> == null:
+      - narrate "<&c>You must specify how much you want to pay."
+      - stop
+    - if <[payee]> == null:
+      - narrate "<&c>You must specify who you want to pay."
+      - stop
```

Comment:
- `.second` also is not a tag, that kind of tag exists for `first` and `last`, you'd want `.get[2]` in this case
- it's better to check if the list of arguments are empty, or comparing it's size as opposed to checking if the first and second argument are invalid, or not
- you'd also optimally want to verify the `amount` is a valid integer
```suggestion
    - if <context.args.size> != 2:
      - inject command_syntax
    - define amount <context.args.get[2]>
    - define payee <context.args.first>
    - if !<[amount].is_integer>:
      - narrate "<&c>You must specify how much you want to pay."
      - stop
```